src/web_app_skeleton/src/serializers/errors/show_serializer.cr

```diff
@@ -1,10 +1,14 @@
 # This is the default error serializer generated by Lucky.
 # Feel free to customize it in any way you like.
 class Errors::ShowSerializer < Lucky::Serializer
-  def initialize(@message : String, @details : String? = nil)
+  def initialize(
+    @message : String,
+    @details : String? = nil,
+    @param : String? = nil
```

Comment:
This is what Stripe has in their error object and I think it is a good default. This way by default you can use `create!/update!` and Lucky will respond with the param that had the problem and what the error message is